src/modules/notebook-panel/NotebookPanel.js

```diff
@@ -47,6 +48,11 @@ const NotebookPanel = (props) => {
     };
   }, [isNotebookPanelVisible]);
 
+  useEffect(() => {
+    dispatch(setSelectedAttributes([]));
```

Comment:
This breaks going directly to a feature detail from the overview page.

Reply:
taken care.

Reply:
Does this mean you are taking care of it or did take care of it? ;) I don't see any changes to this line so if you already took care of it how did you fix it?